lib/compat/wordpress-5.9/class-gutenberg-rest-templates-controller.php

```diff
@@ -579,6 +580,12 @@ public function get_item_schema() {
 					'context'     => array( 'embed', 'view', 'edit' ),
 					'readonly'    => true,
 				),
+				'is_custom'      => array(
```

Comment:
Does this make sense to be only added for `wp_template`? 🤔

Reply:
Currently we add it for both `wp_template` and `wp_template_part` in `get_block_templates`. Property is part of `WP_Block_Template` class, which both  use.

But since this prop isn't helpful for template parts, it makes sense to add it conditionally.

Reply:
Updated in aff6095.